e2e/scripts/st_arrow_altair_global_theme.py

```diff
@@ -1,29 +0,0 @@
-# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
```

Comment:
Why don't we want this test anymore?

Reply:
This was just a temporary feature - before we had the `theme` parameter - that got removed in this PR as a side-effect. It was possible to set the theme via: 

```python
alt.themes.enable("streamlit")
``

But we never advertised it and this has some technical issue related to it, so we introduced the `theme` parameter with our charting commands instead. So this is just removing some unused tech debt.